backend/src/main/java/com/google/rolecall/services/SectionServices.java

```diff
@@ -72,31 +80,73 @@ public Section getSection(Integer id) throws EntityNotFoundException, InvalidPar
    *     or Positions have overlapping orders.
    */
   public Section createSection(SectionInfo newSection) throws InvalidParameterException {
+    Boolean isParentRevelation = newSection.type() == Section.Type.REVELATION;
     Section section = Section.newBuilder()
         .setName(newSection.name())
         .setNotes(newSection.notes())
         .setLength(newSection.length())
+        .setSiblingId(newSection.siblingId())
         .setType(newSection.type())
         .build();
 
+    Integer[] ixArray = new Integer[newSection.positions().size()];
```

Comment:
What does 'ix' prefix represent, could we use a more descriptive var name?

Reply:
I changed the name to 'siblingIndexArray'.